src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__product {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-weight: $font-weight-medium;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0 0 4px;
+    color: $color-dark;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    font-weight: $font-weight-normal;
+    line-height: 14px;
+    color: $color-light;
+  }
+
+  &__stars-review {
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+    width: 100%;
+    flex-grow: 1;
+    height: 16px;
+    margin-bottom: 24px;
+  }
+
+  &__reviews-text {
+    font-size: 10px;
+    font-weight: $font-weight-normal;
+    width: 53px;
+    text-align: right;
+    line-height: 16px;
+    color: $color-dark;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-light;
+  }
+
+  &__sum {
+    font-weight: $font-weight-bold;
+    font-size: 16px;
+    line-height: 18px;
+    color: $color-dark;
+    align-items: flex-end;
+    text-align: right;
+  }
+
+  &__buy-button {
+    font-family: Roboto, sans-serif;
```

Comment:
move to html selector

Reply:
I have already added this in reset.scss:

html {
  font-family: Roboto, sans-serif;
}

However, it doesn't work only for a button.
If I move font-family to html selector in card.scss, it still doesn't work.
The font changes only when I add font-family to .card__buy-button